task.ps1

```diff
@@ -6,6 +6,7 @@ $subnetName = "default"
 $vnetAddressPrefix = "10.0.0.0/16"
 $subnetAddressPrefix = "10.0.0.0/24"
 $publicIpAddressName = "linuxboxpip"
+$domainNameLabel = "my-domain-name"
```

Comment:
Ensure that the `domainNameLabel` is unique within the Azure region to avoid conflicts when creating the public IP address.